README.md

```diff
@@ -43,6 +43,24 @@ tree.on('move', function (node, newParent, previousParent, newIndex, prevIndex)
 })
 ```
 
+Listen for the start of dnd
+```
+tree.on('dndstart', function (movingNode, travelingNode, movingNodeBoundData) {
+  // movingNode -> The node the user clicked and is trying to move
+  // travelingNode -> The node created by the tree which can actually be moved
```

Comment:
I think it would be helpful to starting this with "The DOM node" instead of just "the node"